doc/release_notes_0.2.rst

```diff
@@ -17,8 +17,63 @@ ACRN device model, and documentation.
 Version 0.2 new features
 ************************
 
+VT-x, VT-d
+================
+Based on Intel VT-x virtualization technology, ACRN emulates virtual CPU with core partition and simple schedule.
+VT-d stands for Intel VT for Directed IO, providing capabilities to assign I/O devices to VMs and extending the protection and isolation properties of VMs for I/O operations.
+
+PIC/IOAPIC/MSI/MSI-X/PCI/LAPIC
+================
```

Comment:
The title underline must be at least as long as the title, so add some more ``=====``